sass/mixins/_mixins-master.scss

```diff
@@ -89,90 +89,97 @@
+	html[lang="sr-RS"] #{$wrapper_selector},
+	html[lang="tt-RU"] #{$wrapper_selector},
+	html[lang="uk"] #{$wrapper_selector} {
 	  font-family: 'Helvetica Neue', Helvetica, 'Segoe UI', Arial, sans-serif !important;
 	}
 
 	/* Chinese (Hong Kong) */
-	html[lang="zh-HK"] #{$wrapper_classname} * {
+	html[lang="zh-HK"] #{$wrapper_selector} {
 		font-family: -apple-system, BlinkMacSystemFont, 'PingFang HK', 'Helvetica Neue', "Microsoft YaHei New", STHeiti Light, sans-serif !important;
 	}
 
 	/* Chinese (Taiwan) */
-	html[lang="zh-TW"] #{$wrapper_classname} * {
+	html[lang="zh-TW"] #{$wrapper_selector} {
 		font-family: -apple-system, BlinkMacSystemFont, 'PingFang TC', 'Helvetica Neue', "Microsoft YaHei New", STHeiti Light, sans-serif !important;
 	}
 
 	/* Chinese (China) */
-	html[lang="zh-CN"] #{$wrapper_classname} * {
+	html[lang="zh-CN"] #{$wrapper_selector} {
 		font-family: -apple-system, BlinkMacSystemFont, 'PingFang SC', 'Helvetica Neue', "Microsoft YaHei New", STHeiti Light, sans-serif !important;
 	}
 
 	/* Devanagari */
-	html[lang="bn-BD"] #{$wrapper_classname} *,
-	html[lang="hi-IN"] #{$wrapper_classname} *,
-	html[lang="mr"] #{$wrapper_classname} *,
-	html[lang="ne-NP"] #{$wrapper_classname} * {
+	html[lang="bn-BD"] #{$wrapper_selector},
+	html[lang="hi-IN"] #{$wrapper_selector},
+	html[lang="mr"] #{$wrapper_selector},
+	html[lang="ne-NP"] #{$wrapper_selector} {
 	  font-family: Arial, sans-serif !important;
 	}
 
 	/* Greek */
-	html[lang="el"] #{$wrapper_classname} * {
+	html[lang="el"] #{$wrapper_selector} {
 	  font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif !important;
 	}
 
 	/* Gujarati */
-	html[lang="gu"] #{$wrapper_classname} * {
+	html[lang="gu"] #{$wrapper_selector} {
 	  font-family: Arial, sans-serif !important;
 	}
 
 	/* Hebrew */
-	html[lang="he-IL"] #{$wrapper_classname} * {
+	html[lang="he-IL"] #{$wrapper_selector} {
 	  font-family: 'Arial Hebrew', Arial, sans-serif !important;
 	}
 
 	/* Japanese */
-	html[lang="ja"] #{$wrapper_classname} * {
+	html[lang="ja"] #{$wrapper_selector} {
 	  font-family: -apple-system, BlinkMacSystemFont, "Hiragino Sans", Meiryo, "Helvetica Neue", sans-serif !important;
 	}
 
 	/* Korean */
-	html[lang="ko-KR"] #{$wrapper_classname} * {
+	html[lang="ko-KR"] #{$wrapper_selector} {
 	  font-family: 'Apple SD Gothic Neo', 'Malgun Gothic', 'Nanum Gothic', Dotum, sans-serif !important;
 	}
 
 	/* Thai */
-	html[lang="th"] #{$wrapper_classname} * {
+	html[lang="th"] #{$wrapper_selector} {
 	  font-family: 'Sukhumvit Set', 'Helvetica Neue', helvetica, arial, sans-serif !important;
 	}
 
 	/* Vietnamese */
-	html[lang="vi"] #{$wrapper_classname} * {
+	html[lang="vi"] #{$wrapper_selector} {
 	  font-family: 'Libre Franklin', sans-serif !important;
 	}
+
+	/* Use standard monospace font for all non-english code and pre */
+	html:not([lang^="en-"]) #{$wrapper_selector} code,
+	html:not([lang^="en-"]) #{$wrapper_selector} pre {
+		font-family: monospace;
```

Comment:
I think this needs an `!important` rule in order to override the `!important` stuff above.